extensions/Worker.Extensions.SignalRService/src/Models/SignalRGroupAction.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// An action to manage a group.
+    /// </summary>
+    public sealed class SignalRGroupAction
+    {
+        /// <summary>
+        /// Initializes an instance of a <see cref="SignalRGroupAction"/>.
+        /// </summary>
+        /// <param name="actionType">The action type.</param>
+        public SignalRGroupAction(SignalRGroupActionType actionType)
+        {
+            Action = actionType;
+        }
+
+        /// <summary>
+        /// Sets this field if you want to add a connection to a group or remove it from a group.
+        /// </summary>
+        public string? ConnectionId { get; set; }
+
+        /// <summary>
+        /// Sets this field if you want to add a user to a group or remove it from a group/all groups.
+        /// </summary>
+        public string? UserId { get; set; }
+
+        /// <summary>
+        /// The group to manage. Leave it null if you want to remove a user from all groups.
+        /// </summary>
+        public string? GroupName { get; set; }
+
+        /// <summary>
+        /// The group action type.
+        /// </summary>
+        public SignalRGroupActionType Action { get; set; }
+
+        /// <summary>
+        /// The SignalR endpoints to send to. Leave it null if you want to send to all endpoints.
+        /// You can get the available SignalR endpoints from the <see cref="SignalREndpointsInputAttribute"/> binding.
+        /// </summary>
+        public SignalREndpoint[]? Endpoints { get; set; }
+    }
+
+    /// <summary>
+    /// The type of a group action.
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
Question: Wonder how it will work if customer chooses to override JSON serializer to be JSON.NET.

Reply:
I have tried to change the JSON serializer to Newtonsoft.JSON as the code below and it still works. Maybe the conversion for bindings is not controlled by users. I am not sure about this, maybe we need a dotnet worker expert to confirm this.
```cs
    class Program
    {
        static async Task Main(string[] args)
        {
            var host = new HostBuilder()
                .ConfigureFunctionsWorkerDefaults(o =>
                {
                    var settings = NewtonsoftJsonObjectSerializer.CreateJsonSerializerSettings();
                    settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
                    settings.NullValueHandling = NullValueHandling.Ignore;
                    o.Serializer = new NewtonsoftJsonObjectSerializer(settings);
                })
                .Build();
            await host.RunAsync();
```

Reply:
Is there any way to avoid customers changing the serialization and deserialization of some field of my POJO? For example, I want to make the enum `SignalRGroupAction` serialized to string instead of number no matter how customers set their object serializer. But for the `Arguments` field in `SignalRInvocationContext`, `SignalRMessageAction`, I want to let the customers to control the serialization/deserialization.

@fabiocav

Reply:
This is not a blocker to merge this change. Something we should validate though.

Reply:
To add context. No, today, the configured serializer will be used by default. We're adding features that will give binding authors better control of this, but in the meantime, I agree with @kshyju , we don't need to block, but it might be something worth documenting.